homeassistant/components/roon/media_player.py

```diff
@@ -374,6 +374,9 @@ def media_previous_track(self):
     def media_seek(self, position):
         """Send seek command to device."""
         self._server.roonapi.seek(self.output_id, position)
+        # Seek doesn't cause an async update - so force one
+        self._media_position = position
+        self.async_write_ha_state()
```

Comment:
We can't use the async api within sync context. Use `self.schedule_update_ha_state`.
```suggestion
        self.schedule_update_ha_state()
```